docs/dev/table/functions/udfs.md

```diff
@@ -1416,15 +1862,6 @@ tab
 </div>
 
 
-{% top %}
-
-Best Practices for Implementing UDFs
-------------------------------------
-
-The Table API and SQL code generation internally tries to work with primitive values as much as possible. A user-defined function can introduce much overhead through object creation, casting, and (un)boxing. Therefore, it is highly recommended to declare parameters and result types as primitive types instead of their boxed classes. `Types.DATE` and `Types.TIME` can also be represented as `int`. `Types.TIMESTAMP` can be represented as `long`. 
-
-We recommended that user-defined functions should be written by Java instead of Scala as Scala types pose a challenge for Flink's type extractor.
-
```

Comment:
Its below the diff, but can you change the code example in "Integrating UDFs with the Runtime" to use `TableEnvironment` instead of `BatchTableEnvironment`?